Makefile

```diff
@@ -132,6 +149,22 @@ docker-build-collector:
 docker-push-collector:
 	docker push ${COLLECTOR_IMG}
 
+docker-build-eraser-windows:
+	docker buildx build $(_CACHE_FROM) $(_CACHE_TO) --platform="windows/amd64" --output=type=docker --build-arg OSVERSION=${OSVERSION} -t ${REGISTRY}/${REPO}-windows-${OSVERSION}-amd64:${VERSION} -f windows.Dockerfile .
+
+docker-push-windows:
+	docker push ${REGISTRY}/${REPO}-windows-${OSVERSION}-amd64:${VERSION}
```

Comment:
@vyta is this going to work since we can't load windows images in linux?

Reply:
Ah, you're right! Great catch. What's the best way to push the windows images? I suppose one option would be to update the output type of the buildx build to registry, but it appears registry login doesn't happen until after build step.

Reply:
that's right. this is the part that gets a little tricky, we need to push to a staging registry. for other projects, we use ADO to run this on our own agents/registry but we don't have that infra set up yet for eraser.

Reply:
Gotcha. Perhaps we leave off the windows release in this PR and create a separate issue to track that work?

Reply:
SGTM. I am okay scoping it down to "support building windows images" and we can figure out the push/release at a later time once we have the infra set up for it